boards/arm/nucleo_f302r8/pinmux.c

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2018 Diego Asanza
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <kernel.h>
+#include <device.h>
+#include <init.h>
+#include <pinmux.h>
+#include <sys_io.h>
+
+#include <pinmux/stm32/pinmux_stm32.h>
+
+/* pin assignments for NUCLEO-F302R8 board */
+static const struct pin_config pinconf[] = {
+#ifdef CONFIG_UART_STM32_PORT_1
+	{STM32_PIN_PA9,  STM32F3_PINMUX_FUNC_PA9_USART1_TX},
+	{STM32_PIN_PA10, STM32F3_PINMUX_FUNC_PA10_USART1_RX},
+#endif	/* CONFIG_UART_STM32_PORT_1 */
+#ifdef CONFIG_I2C_1
+	{STM32_PIN_PB8, STM32F3_PINMUX_FUNC_PB8_I2C1_SCL},
+	{STM32_PIN_PB9, STM32F3_PINMUX_FUNC_PB9_I2C1_SDA},
+#endif /* CONFIG_I2C_1 */
+#ifdef CONFIG_UART_STM32_PORT_2
+	{STM32_PIN_PA2, STM32F3_PINMUX_FUNC_PA2_USART2_TX},
+	{STM32_PIN_PA3, STM32F3_PINMUX_FUNC_PA3_USART2_RX},
+#endif	/* CONFIG_UART_STM32_PORT_2 */
+#ifdef CONFIG_SPI_1
```

Comment:
Did you test SPI on this SoC? I checked the HAL files and it doesn't seem to support it?
@erwango Could you also check it?

Reply:
You are right. This soc does not have spi 1. Spi 1 removed in 3826cb0